mathics/builtin/no_meaning.py

```diff
@@ -1,42 +1,51 @@
 # -*- coding: utf-8 -*-
-# Note: this will be redone soon to pull no-meaning operator information from operators.yml out
-# of the MathicsScanner project.
 """
 Operators without Built-in Meanings
 
 Not all operators recognized by the Mathics3 are associated with functions that have built‐in meanings.
-You can use these operators as a way to build up your own notation within the Mathics3.
+You can use these operators as a way to build up your own notation within Mathics3.
 """
 
-from mathics.core.attributes import A_NO_ATTRIBUTES
-from mathics.core.builtin import InfixOperator
+from inspect import getmembers, isclass
+from sys import modules
 
+from mathics.core.builtin import OPERATOR_DATA, NoMeaningInfixOperator
 
-class Star(InfixOperator):
-    r"""
-    Star <url>
-    :WML link:
-    https://reference.wolfram.com/language/ref/Star.html</url>
+# Note: classes in this file must *only* be "no-meaning"
+# builtin operator classes.
 
-    <dl>
-      <dt>'Star[$x$, $y$, ...]'
-      <dd>displays $x$ ⋆ $y$ ⋆ ...
-    </dl>
 
-    >> Star[x, y, z]
-     = x ⋆ y ⋆ z
+class Because(NoMeaningInfixOperator):
+    r"""This text is replaced! But it needs to be here for documentation detection."""
 
-    >> a \[Star] b
-     = a ⋆ b
-    """
 
-    attributes = A_NO_ATTRIBUTES
-    default_formats = False  # Don't use any default format rules. Instead, see below.
-    formats = {
-        (("InputForm", "OutputForm", "StandardForm"), "Star[args__]"): (
-            'Infix[{args}, "⋆"]'
-        ),
-    }
+class Cap(NoMeaningInfixOperator):
+    r"""This text is replaced! But it needs to be here for documentation detection."""
 
-    operator = "⋆"  # \u22C6
-    summary_text = "star symbol"
+
+class CenterDot(NoMeaningInfixOperator):
+    r"""This text is replaced! But it needs to be here for documentation detection."""
+
+
+class Star(NoMeaningInfixOperator):
+    r"""This text is replaced! But it needs to be here for documentation detection."""
+
+
+# Generate Builtin No-meaning Builtin Infix operators, using
+# the Operator name and Operator Unicode found by reading
+# the operators JSON file.
+for name, operator_class in getmembers(modules[__name__]):
```

Comment:
Would be it possible to check if we have Builtin symbols for all the "no meaning infix operators" existent in the table? (Not here, but in the consistency tests)

Reply:
Patience. I am about to commit this.